addons/gdUnit4/src/ui/parts/InspectorTreeMainPanel.gd

```diff
@@ -14,7 +14,7 @@ signal run_testsuite
 @onready var _context_menu_debug := $contextMenu/items/debug
 
 # tree icons
-@onready var ICON_SPINNER = load("res://addons/gdUnit4/src/ui/assets/spinner.tres")
+@onready var ICON_SPINNER = preload("res://addons/gdUnit4/src/ui/assets/spinner.tres")
```

Comment:
Can you explain in more detail why this preloading should fix the UUI problems?
As I understand it, it makes no difference when the image is loaded.
And since it is initially loaded only once, it shouldn't matter if you use load or preload.

Reply:
I forgot to mention the remark of KoBeWi in https://github.com/godotengine/godot/issues/72273#issuecomment-1454804342

> It might have something to do with the fact that AnimatedTexture is not a "real" texture (i.e. it's not imported etc.).

That triggered my preload 'quest'